### ISSUE TITLE
Inline for Rich.Embed not working

### ISSUE THREAD

**Issue by KannaDev** [discordjs/discord.js#2799]
**Please describe the problem you are having in as much detail as possible:**
Previously, in discord v11.3.2 inline worked great, but after I updated it to 11.4 it stopped working.

**Include a reproducible code sample here, if possible:**
```js
let lifeEmbed = new Discord.RichEmbed()
          .setTitle("## FORTNITE LIFETIME STATS ##")
          .setThumbnail(
            "https://blog.lifetime.com/imagecache/Blog/Generic%20Lifetime%20Banner%20Blog.png"
          )
          .setDescription(`Lifetime stats for ${data.username}`)
          .setColor("#42b6f4")
          .addField("Wins", lifeWins, true)
          .addField("Kills", lifeKills, true)
          .addField("K/D", lifeKd, true)
          .addField("Matches Played", lifeMatches, true)
          .addField("Score", lifeScore, true)
          .addField("Win Percentage", lifeWinPercent, true);

        message.channel.send(lifeEmbed);
```
That code previously generated this [response](https://imgur.com/AwXxyCD), but after the update is producing this [response](https://imgur.com/En3qR4O)

**Further details:**
- discord.js version: 11.4.2
- Node.js version: v8.11.2
- Operating system: MAC OS X, Ubuntu 16.04
- Priority this issue should have – please be realistic and elaborate if possible: Medium

- [ ] I found this issue while running code on a __user account__
- [ ] I have also tested the issue on latest master, commit hash:

**Comment by SpaceEEC** [comment]
That bug had been introduced here https://github.com/discordjs/discord.js/commit/87b4b23596796fc351aafa87c497c601e3a8f306#diff-dd560c495a00593ea828a458e5e13b18R251, which was then released with 11.4.

After it got discovered it was fixed with https://github.com/discordjs/discord.js/commit/7126cade4559233ebf96ee3ba78bb27da1923596 and released with 11.4.1.

**Comment by KannaDev** [comment]
@SpaceEEC Thank you, turns out it was still on v11.4.0. 
Worked after the update.
